bin/health-check.rb

```diff
@@ -43,7 +43,6 @@ def error(message)
 end
 
 # When authenticating as a host, ensure that credentials can access host resource.
-# Conjur v4 grant this privilege automatically, so we only check for v5.
 if ConjurClient.version == 5
```

Comment:
This is outside of your change, but this `if ConjurClient.version == 5` should have an `else ... error("Conjur version <VERSION> is not supported") to avoid silent failures.

Reply:
there is a comparable check in the service broker health check which also runs on startup - it would be a nice to have, but I believe it is already covered elsewhere.

Reply:
Other than this ^^^, it looks awesome.

Reply:
I think we should delete the conditional then, if it's redundant. Also, based on the comment that's being deleted
this might be considered v4 support cruft (i.e. it's probably unlikely that v5+ will bring back automatic privilege granting).

Reply:
removed check 👍